### DISCUSSION TITLE
gpu based blur filter without cropping edges

### DISCUSSION THREAD

**Discussion by vondiplo** [fabricjs/fabric.js#7989] [Q&A]
Hello,

I've played along with the built-in image filters listed here -  http://fabricjs.com/image-filters.
I noticed that the blur filter is cropped to match the original size of the image.
What I'd like to achieve is a larger version of the image with soft edges and gradual alpha channel decay.
Is it possible to modify the blur filter or create a new one that returns an image of a different size?

for example, I'd like to achieve the following -
<img width="694" alt="Screen Shot 2022-06-02 at 6 48 49 PM" src="https://user-images.githubusercontent.com/1870445/171670716-499653be-f570-4f83-a97f-1a242abbb2d3.png">

from this original image:
![pug](https://user-images.githubusercontent.com/1870445/171670821-35fc789b-949e-4390-bcc0-62210639d6a1.png)
 
Instead I get this: 
<img width="521" alt="Screen Shot 2022-06-02 at 6 52 18 PM" src="https://user-images.githubusercontent.com/1870445/171671222-e240a0b6-34f7-4751-8df9-96a8d3ab50d6.png">

Thank you

**Comment by vondiplo** [comment]
any chance to get an answer from someone ?

**Comment by mattevans-dev** [comment]
Also very interested in this. Is this possible in the current system @asturur @ShaMan123 ? E.g applying a filter and not being cropped by it's bounding box. Useful for filters such as blur or repeating edges.
